templates/agent/src/components/ChatBox.tsx

```diff
@@ -33,6 +34,7 @@ const messages = new Map([
 export default function ChatBox({refetch}: Props) {
 	const [prompt, setPrompt] = useState('')
 	const [loading, setLoading] = useState(false)
+	const [bot, setBot] = useState<string>('gpt-3.5-turbo')
```

Comment:
Can typecast this to `keyof typeof models` by exporting `models` [here](https://github.com/RubricLab/create-rubric-app/pull/18/files#diff-61af52ad15690dacd8b92129306efa345e2f1644d5039649de2ebd35f894be56R8)

Even better, add a file `/utils/types.ts` and `export type Model = keyof typeof models` then import `Model` here